app/javascript/components/pages/admin/users-form/container.jsx

```diff
@@ -123,9 +123,32 @@ const Container = ({ mode }) => {
     formState: { dirtyFields }
   } = formMethods;
 
+  function replaceNullWithBlank(obj) {
```

Comment:
Move this function into `app/javascript/components/pages/admin/users-form/utils.js`